src/SFA.DAS.EmployerDemand.Api/Controllers/DemandController.cs

```diff
@@ -79,7 +80,21 @@ public async Task<IActionResult> CreateDemand(CourseDemandRequest request)
                 _logger.LogError(e,"Unable to create course demand");
                 return new StatusCodeResult((int) HttpStatusCode.InternalServerError);
             }
-            
+        }
+
+        [HttpGet]
+        [Route("get-aggregated-demands")]
+        public async Task<IActionResult> GetAggregatedCourseDemandList()
```

Comment:
Seeing that this will need to accept the `ukprn` we could probably change the name of the action/route to reflect that